_data/sites.json

```
@@ -7900,6 +7900,14 @@
             "help.glassdoor.com"
         ]
     },
+    {
+        "name": "Gleam.io",
+        "url": "https://gleam.io/faq/accounts-and-billing/account/how-do-i-delete-my-account",
```

```suggestion
        "url": "https://gleam.io",
```

If the profile URL is specific, put the URL to the homepage and leave the notes directing people to the account settings. I think the FAQ will just confuse them